flow-typed/most.js

```diff
@@ -0,0 +1,414 @@
+declare module "most" {
+
+  declare type SeedValue<S, V> = { seed: S, value: V };
+  declare type TimeValue<V> = { time: number, value: V };
+
+  declare interface Generator<A, B, C> {}
+  declare interface Iterable<A> {}
+
+  declare type CreateGenerator<A> = (...args: Array<any>) => Generator<A|Promise<A>, any, any>;
+
+  declare export interface Sink<A> {
+    event(time: number, value: A): void;
+    end(time: number, value?: A): void;
+    error(time: number, err: Error): void;
+  }
+
+  declare export interface Task {
+    run(time: number): void;
+    error(time: number, e: Error): void;
+    dispose(): void;
+  }
+
+  declare export interface ScheduledTask {
+    task: Task;
+    run(): void;
+    error(err: Error): void;
+    dispose(): void;
+  }
+
+  declare export interface Scheduler {
+    now(): number;
+    asap(task: Task): ScheduledTask;
+    delay(task: Task): ScheduledTask;
+    periodic(task: Task): ScheduledTask;
+    schedule(delay: number, period: number, task: Task): ScheduledTask;
+    cancel(task: Task): void;
+    cancelAll(predicate: (task: Task) => boolean): void;
+  }
+
+  declare export interface Disposable<A> {
+    dispose(): void | Promise<A>;
```

Comment:
There is no good reason, just is what it is. :+1: for simplifying to `Promise<any>` with not template type on Disposable. We should do that in the TS typings as well.